presets/tuning/text-generation/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Core Dependencies
 transformers==4.41.2
-torch==2.2.0
+torch==2.4.0
```

Comment:
Why editing working torch version for fine tuning?

Reply:
By mistake, will fix